flowetl/etl/etl/dag_task_callable_mappings.py

```diff
@@ -79,6 +84,13 @@
     "extract": partial(PostgresOperator, postgres_conn_id="flowdb"),
     "transform": partial(PostgresOperator, postgres_conn_id="flowdb"),
     "load": partial(PostgresOperator, postgres_conn_id="flowdb"),
+    "postload": partial(
```

Comment:
Would it make more sense for the post load informative queries to be a subdag? That way you can specify lots of independent ones?